PythonTool/GraveIdentifyAndRemove.py

```diff
@@ -147,10 +147,9 @@ def remove_graveyard(input_file, output_file = None, print_handle = None):
     if output_file is not None:
         mb.write_file(output_file, output_sets=groups_to_write)
     else:
-        input_array = input_file.split("/")
-        input_file = input_array[-1]
-        base_file_name = input_file[:-4]
-        output_file = base_file_name + "_no_grave.h5m"
+        input_list = input_file.split("/")
```

Comment:
These updates got me thinking...

  - removing the last 4 characters is nice, but what if a file extension we want to support has more than 4 characters?

  - splitting on "." is nice too, but what if a user likes to have "." as part of their filename? It isn't recommended typically, but it happens. 

At any rate, there are some options as part of `os` (which we currently import anyway). See [here](https://stackoverflow.com/questions/678236/how-to-get-the-filename-without-the-extension-from-a-path-in-python)

**Not** making a request for a change here. Just passing along some info if you're interested. :smile:

Reply:
That's probalby the best method presented yet. `os` just has some standard options that we might find useful without having to worry too much about the specific implementation. I'm cool w/ whatever here.